src/core/crypto.h

```diff
@@ -439,6 +439,18 @@ QuicCryptoReNegotiateAlpn(
         const uint8_t* AlpnList
     );
 
+//
+// Fill keys for offloading
+//
+_Success_(return==TRUE)
+BOOLEAN
+QuicPacketKeyCreateOffload(
```

Comment:
This should be declared in quic_tls.h instead and probably called `QuicTlsPopulateKeyOffload` or something similar.